UI/script.js

```diff
@@ -1,561 +1,852 @@
-      photoLink: 'https://sila.by/img/catalog2015/ntpk/tolet111378.jpg',
-      hashTags: ['Техника'],
-      discount: '20%',
-      rating: 1,
-      validUntil: new Date('2022-03-21T12:00:00'),
-      reviews: ['Cool!'],
-    },
-
-    {
-      id: '2',
-      description: 'Скидка на столы',
-      createdAt: new Date('2021-02-21T12:00:00'),
-      link: 'https://tables.com',
-      vendor: 'Best Tables',
-      hashTags: ['Мебель'],
-      discount: '25%',
-      validUntil: new Date('2021-05-21T12:00:00'),
-      rating: 1.1,
-      reviews: ['Cool!'],
-    },
-
-    {
-      id: '3',
-      description: 'Скидка на тетради',
-      createdAt: new Date('2021-03-15T12:00:00'),
-      link: 'https://m4.by',
-      vendor: 'M4.BY',
-      hashTags: ['Школа'],
-      discount: '10%',
-      validUntil: new Date('2021-04-21T12:00:00'),
-      rating: 1.2,
-      reviews: ['Cool!'],
-    },
-    {
-      id: '4',
-      description: 'Скидка на все игры',
-      createdAt: new Date('2021-03-14T12:00:00'),
-      link: 'https://store.steampowered.com/?l=russian',
-      vendor: 'Steam',
-      hashTags: ['Компьютер', 'Игры'],
-      discount: '5%',
-      validUntil: new Date('2021-05-21T12:00:00'),
-      rating: 1.3,
-      reviews: ['Cool!'],
-    },
-
-    {
-      id: '5',
-      description: 'Скидка на цветы',
-      createdAt: new Date('2021-02-13T12:00:00'),
-      link: 'https://flowers.by',
-      vendor: 'Мир цветов',
-      photoLink: 'https://cdn2.flowwow.com/data/blog/1562599250_29351868.jpg',
-      hashTags: ['Растения'],
-      discount: '30%',
-      validUntil: new Date('2022-03-21T12:00:00'),
-      rating: 1.4,
-      reviews: ['Cool!'],
-    },
-
-    {
-      id: '6',
-      description: 'Скидка на продукцию Xiaomi -10%',
-      createdAt: new Date('2021-01-01T11:00:00'),
-      link: 'https://5element.by',
-      vendor: '5 элемент',
-      hashTags: ['Техника'],
-      discount: '10%',
-      validUntil: new Date('2021-03-21T12:00:00'),
-      rating: 1.5,
-      reviews: ['Cool!'],
-    },
-
-    {
-      id: '7',
-      description: 'Скидка на обои',
-      createdAt: new Date('2020-03-21T12:00:00'),
-      link: 'https://wallpapers.by',
-      vendor: 'Мир обоев',
-      hashTags: ['Товары для дома'],
-      discount: '15%',
-      validUntil: new Date('2021-05-21T12:00:00'),
-      rating: 1.6,
-      reviews: ['Cool!'],
-    },
-
-    {
-      id: '8',
-      description: 'Скидка на подушки -5%',
-      createdAt: new Date('2019-03-21T12:00:00'),
-      link: 'https://goodsforsleep.by',
-      vendor: 'Товары для сна',
-      hashTags: ['Сон'],
-      discount: '5%',
-      validUntil: new Date('2021-05-21T12:00:00'),
-      rating: 1.7,
-      reviews: ['Cool!'],
-    },
-
-    {
-      id: '9',
-      description: 'Скидка на все моющие средства до конца года -10%',
-      createdAt: new Date('2021-02-26T12:00:00'),
-      link: 'https://ostrov-chistoty.by/',
-      vendor: 'Остров чистоты',
-      photoLink:
+
```

Comment:
Let's create a separate file for each class and move all testing code to one more separate file - just for clean code sake